File: benchmarks/yass/frequencies.jl
using Pkg
Pkg.activate("./")

using YetAnotherSimulationSuite

i = ARGS[1]

bdys = readSystem("../xyzfiles/$(i)au_opt.xyz")

d = Dict(
  "epsilon" => 0.2297,
  "sigma" => 2.95,
  "rs" => 19.0,
  "rc" => 20.0
)

calc = LJ(d)

v,m = getHarmonicFreqs(calc, bdys)

File: benchmarks/ase/frequencies.py
from ase.calculators.lj import LennardJones
from ase.io import read
from ase.vibrations import Vibrations
import sys

i = sys.argv[1]

bdys = read(f"../xyzfiles/{i}au_opt.xyz")

calc = LennardJones(sigma=2.95, epsilon=0.2297, ro=19.0, rc=20.0)

bdys.calc = calc

vib = Vibrations(bdys)
vib.run()

